fe/fe-core/src/main/java/com/starrocks/credential/CloudCredential.java

```diff
@@ -40,4 +41,6 @@ public interface CloudCredential {
 
     // Generate unique credential string, used in FileSystem cache
     String getCredentialString();
+
+    FileStoreInfo toFileStore();
```

Comment:
Add some comment about it.
`toFileStoreInfo()` ?

Reply:
FileStoreInfo toFileStoreInfo();